packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -193,11 +154,30 @@ class ScrollBehavior {
   /// Applies a [GlowingOverscrollIndicator] to the child widget on
   /// [TargetPlatform.android] and [TargetPlatform.fuchsia].
   Widget buildOverscrollIndicator(BuildContext context, Widget child, ScrollableDetails details) {
-    // TODO(Piinks): Move implementation from buildViewportChrome here after
-    //  deprecation period
-    // When modifying this function, consider modifying the implementation in
-    // the Material and Cupertino subclasses as well.
```

Comment:
Should we keep this comment? That's still true and relevant, right?

Reply:
Oh yeah definitely. Let me update that across subclasses as well while we are here. Thanks!